decksite/templates/cardstable.mustache

```diff
@@ -6,23 +6,33 @@
             <th class="n" data-sortInitialOrder="desc"># Decks</th>
             <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">Record</th>
             <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">Win %</th>
-            <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">①</th>
-            <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">⑧</th>
-            <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">5–0s</th>
+            <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">
+                <abbr title="Tournament wins">①</abbr>
+            </th>
+            <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">
+                <abbr title="Tournament top-8s">⑧</abbr>
+            </th>
+            <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">
+                <abbr title="League 5-0 runs">5–0s</abbr>
+            </th>
+            <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">
+                <abbr title="Number of decks with card in the sideboard"> # SBs</abbr>
+            </th>
             {{! Empty column to hold the toggle when footable kicks in.}}
             <td></td>
         </tr>
     </thead>
     <tbody>
         {{#cards}}
-            <tr>
+            <tr class="cardrow" id="cardrow-{{name}}">
```

Comment:
I’m not a big fan of ids in html in general. Things you expect to remain unique have a tendency to become ununique. In this case it’s pretty easy to imagine a page with two cardtables on it where the same card appears in both tables.  Can we make this a class?  I’d also sanitize the name down to lowercase letters and hyphens only personally but that’s less important. Hopefully gzip means this doesn’t really add much page weight to something like /seasons/all/cards but might be worth a spot check.

Reply:
Changed to not using id. Everything is stuffed into classes, except I put the card name into a data attribute, which seemed like a fancy thing that I just learned about.

Reply:
The size of the html doc changes by about 10%, and pd.js is about 30% bigger: https://imgur.com/a/7LN6WyY